tests/models/test_sasrec.py

```diff
@@ -0,0 +1,556 @@
+import typing as tp
+from typing import List
+
+import numpy as np
+import pandas as pd
+import pytest
+import torch
+from pytorch_lightning import seed_everything
+
+from rectools.columns import Columns
+from rectools.dataset import Dataset, IdMap, Interactions
+from rectools.models.sasrec import SASRecDataPreparator, SASRecModel, SequenceDataset
+from tests.models.utils import assert_second_fit_refits_model
+from tests.testing_utils import assert_id_map_equal, assert_interactions_set_equal
+
+
+@pytest.mark.filterwarnings("ignore::pytorch_lightning.utilities.warnings.PossibleUserWarning")
```

Comment:
Could you please add comments for this and the next line? (What the warnings are and why do we ignore them)

Reply:
fixed

Reply:
It's quite obvious that it's a pytorch_lightning warning, could you add some details? And also some short explanation why we have this warning and why we ignore it

Reply:
removed these filterwarnings. There is one left for class TestSASRecDataPreparator